src/wp-includes/class-wp-block-bindings-processor.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+/**
+ * WP_Block_Bindings_Processor class.
```

Comment:
This has `@access private`, but it may be worth noting early and clearly in this description that the class is only for use by WordPress core and usage by extenders is not supported and likely to break.

Is there any way to avoid adding the file and class? The HTML API should publicly provide the necessary functionality someday.

Reply:
The only caveat of having a final class is that we won't be able to extend it from Gutenberg in future developments. Meaning we will have to always combine a WordPress 'develop' version with Gutenberg in case we need to work both with Core functions and editor features.

Reply:
> Is it possible implement this with an anonymous class in the function that processes block bindings?

Probably 🤔 I can give that a try.